js/views/modals/editListing/EditListing.js

```diff
@@ -553,6 +706,8 @@ export default class extends BaseModal {
     } else {
       // client side validation failed
       this.$saveButton.removeClass('disabled');
+      console.log('no soup for you');
+      window.soup = this.model;
```

Comment:
lol... yeah that's just some debugging stuff I need to remove. I expose the model globally so that I could easily see the validation errors, since not all of them make it to the UI (some only happen if the dev messes up and isn't providing the data in the right format).

Good catch!